fluent/syntax/ast.py

```diff
@@ -73,7 +73,7 @@ def visit(value):
 
         return fun(node)
 
-    def equals(self, other, with_spans=False):
+    def equals(self, other, ignored_fields=['span']):
```

Comment:
It's only needed when you mutate the list inside of the function. The reason why this is tricky is that the default arguments are parsed once when the function is defined. If you then mutate them in the function's body, you end up mutating them for all future calls, too.

The code here doesn't mutate `ignore_fields` at any point.